.gitignore

```diff
@@ -47,8 +47,19 @@ Temporary Items
 .apdisk
 
 node_modules/*
+test/data/angularjs/node_modules/*
```

Comment:
Got failed on yarn/npm install because didnt fin this folders

Reply:
sh: line 0: cd: ./test/data/angularjs: No such file or directory
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! testcafe-angular-selectors@0.4.0 install-angularjs-deps: `cd ./test/data/angularjs && npm i`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the testcafe-angular-selectors@0.4.0 install-angularjs-deps script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

Reply:
@nlevman

Thank you. I fixed it in `testcafe-angular-selectors@0.4.1`. Please update your version.